test/controllers/users.js

```diff
@@ -183,4 +197,124 @@ describe('users controller', () => {
       });
     });
   });
+  describe(`${LIST_USERS_LABEL} GET`, () => {
+    beforeEach('populating database with users', done => {
+      mockUser(1)
```

Comment:
do you need in order? if not, you can use Promise.all

Reply:
👍